### PR TITLE
Enhance Swimm UI

### PR DESCRIPTION
**Pull request by talkor**
- Added testing
- Added Vuepress with basic pages
- Minor improvements

### CODE REVIEW

File: docs/.vuepress/styles/palette.styl [hunk 1]
```diff
@@ -0,0 +1,13 @@
+$accentColor = #1847de
```
Comment (yhamry) [inline docs/.vuepress/styles/palette.styl:1]: File name is on purpose ?

File: docs/components/action.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Action
```
Comment (yhamry) [inline docs/components/action.md:1]: Maybe add all examples from original.. with the variant, etc.

**Review by yhamry — APPROVED** [review]
Looking good!💪🏽

File: docs/.vuepress/styles/palette.styl (hunk 1 above)
Comment (talkor) [reply docs/.vuepress/styles/palette.styl:1]: Yeah that's a Stylus file, this is how you customize variables in Vuepress

File: docs/components/action.md (hunk 2 above)
Comment (talkor) [reply docs/components/action.md:1]: Cool will add

**Pull request merged**